Source/WebCore/html/parser/HTMLSrcsetParser.cpp

```diff
@@ -55,33 +53,33 @@ enum DescriptorTokenizerState {
 };
 
 template<typename CharType>
-static void appendDescriptorAndReset(const CharType*& descriptorStart, const CharType* position, Vector<StringView>& descriptors)
+static void appendDescriptorAndReset(std::span<const CharType>& descriptorStart, const CharType* position, Vector<StringView>& descriptors)
```

Comment:
Maybe this should take unsigned as position?

Reply:
That would force the 4 call sites to convert the pointer into an unsigned instead of doing it only once here inside the implementation. For this reason, I felt it was better to keep taking a pointer.